ngraph/frontend/tensorflow/src/op/arg_min_max.cpp

```diff
@@ -14,46 +14,43 @@ namespace tf {
 namespace op {
 
 OutputVector TranslateArgMinMax(const NodeContext& node, std::string mode) {
```

Comment:
please rename `translate_argmin_max`

Reply:
I propose to fix it in the next PR

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/8477